pynetbox/core/api.py

```diff
@@ -30,42 +30,60 @@ class Api:
     you can specify which app and endpoint you wish to interact with.
 
     Valid attributes currently are:
-        * circuits
-        * core (NetBox 3.5+)
-        * dcim
-        * extras
-        * ipam
-        * tenancy
-        * users
-        * virtualization
-        * vpn (NetBox 3.7+)
-        * wireless
-
-    Calling any of these attributes will return
-    :py:class:`.App` which exposes endpoints as attributes.
-
-    **Additional Attributes**:
-        *  **http_session(requests.Session)**:
-                Override the default session with your own. This is used to control
-                a number of HTTP behaviors such as SSL verification, custom headers,
-                retires, and timeouts.
-                See `custom sessions <advanced.html#custom-sessions>`__ for more info.
-
-    :param str url: The base URL to the instance of NetBox you
-        wish to connect to.
-    :param str token: Your NetBox token.
-    :param bool,optional threading: Set to True to use threading in ``.all()``
-        and ``.filter()`` requests.
-    :raises AttributeError: If app doesn't exist.
-    :Examples:
-
-    >>> import pynetbox
-    >>> nb = pynetbox.api(
-    ...     'http://localhost:8000',
-    ...     token='d6f4e314a5b5fefd164995169f28ae32d987704f'
-    ... )
-    >>> list(nb.dcim.devices.all())
-    [test1-leaf1, test1-leaf2, test1-leaf3]
+    * circuits
+    * core (NetBox 3.5+)
+    * dcim
+    * extras
+    * ipam
+    * tenancy
+    * users
+    * virtualization
+    * vpn (NetBox 3.7+)
+    * wireless
```

Comment:
These aren't rendering as a lists. I think you need another line break before the first item.